doc/news/changes.h

```diff
@@ -669,6 +669,12 @@ inconvenience this causes.
   <br>
   (Matthias Maier, 2014/12/29)
   </li>
+
+  <li> Improved: Used a variety of strategies to silence compiler warnings
+  about unused variables and unused parameters.
```

Comment:
Make it a complete sentence with a subject, e.g., "deal.II now uses a variety..."